src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,21 @@
+spring:
+  application:
+    name: budgetControl
```

Comment:
In Spring Boot, the application.yaml (or application.properties) file serves as the base configuration for your application. Profile-specific configuration files like application-dev.yaml, application-test.yaml, etc., are intended to override or supplement the base configuration for specific environments.​ 

There's no need to duplicate all properties across these files. Spring Boot automatically merges configurations, using the base application.yaml as the default and applying overrides from the active profile's configuration file.
```suggestion
```